packages/bpk-component-sliding-drawer/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "bpk-component-sliding-drawer",
```

Comment:
I propose `bpk-component-drawer`, sliding is part of it's behaviour and as such Im not sure it should be in the name. I.e. what happens when we wanna make it fade in as opposed to sliding?

Reply:
Perhaps @jamesf3rguson can weigh in.